### ISSUE TITLE
Create an English version of the documentation.

### ISSUE THREAD

**Issue by ArslanYM** [eryajf/HowToStartOpenSource#32]
Creating an English version of the documentation would help reach a wider audience and help people like myself to learn about open source and contribute more.

**Comment by eryajf** [comment]
Great advice, but I'm sorry, my English is very poor. People who are proficient in English and are interested in it are required to participate.

**Comment by ArslanYM** [comment]
> Great advice, but I'm sorry, my English is very poor. People who are proficient in English and are interested in it are required to participate.

Yeah sure, Can I start working on it?

**Comment by eryajf** [comment]
Of course, you are very welcome, but there are still some questions before you start. As far as I know, this blog topic has some problems with multilingual support, and I need to do some work in this area.

Of course you can start your work, the two do not conflict.

**Comment by ArslanYM** [comment]
I have already created a English version of the Readme file.  Can you guide me on what the branch name should be called and how should i proceed? @eryajf

**Comment by eryajf** [comment]
You can continue your work first, give me some time, let me make some adjustments to the project to see how to support multiple languages, at that time I will introduce the use of multiple languages

**Comment by eryajf** [comment]
I need to ask, are you going to translate the documents on your side? if so, I need to modify the blog program. If not, then I will ignore the matter for the time being

**Comment by ArslanYM** [comment]
Yes, I will start translating as much as I can on my side, I will also ask my friends to start contributing to this repo as it is very helpful

**Comment by eryajf** [comment]
Okay, I see. I'll make adjustments as soon as possible.

**Comment by eryajf** [comment]
After a day of research, the online platform now supports language switching between Chinese and English.

If you want to translate, you can start here.：https://github.com/eryajf/HowToStartOpenSource/tree/main/docs/en/views

**Comment by ArslanYM** [comment]
@eryajf  On it. Again , thankyou so much for letting me work on this. Youre amazing!!

**Comment by ArslanYM** [comment]
@eryajf i accidentally pushed my new changes to the same branch of my forked repo. should i go for compare and pull request?

**Comment by eryajf** [comment]
I'm not sure if I understand what you mean correctly.

What you are doing now should be correct, no doubt, look at our documentation.

https://eryajf.github.io/HowToStartOpenSource/en/views/01-basic-content/03-collaborative-development-process.html

Fork the project to your account, then modify it and submit it. If there is no problem, you can come to me to create a merge request.

If you have any questions, you are welcome to communicate again
